epowcore/pandapower/pandapower_model.py

```diff
@@ -0,0 +1,447 @@
+"""pandapower_model module providing the PandapowerModel class used to create
+the converted pandapower network.
+"""
+
+from dataclasses import dataclass
+import math
+from math import pi
+
+import pandapower
+from numpy import nan
```

Comment:
Typical numpy import is `import numpy as np`. That way, you always know that `np.nan` comes from numpy. And it's still short ;)

You could also change `pi` to `math.pi` to clean that up a bit.